extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -203,8 +231,35 @@ export function getConnectionProfile(
 	};
 }
 
-export async function collectSourceDatabaseTableInfo(sourceConnectionId: string, sourceDatabase: string): Promise<TableInfo[]> {
-	const ownerUri = await azdata.connection.getUriForConnection(sourceConnectionId);
+export async function getSourceConnectionString(): Promise<string> {
+	return await azdata.connection.getConnectionString((await getSourceConnectionProfile()).connectionId, true);
+}
+
+export async function getTargetConnectionString(
+	serverName: string,
+	azureResourceId: string,
+	username: string,
+	password: string): Promise<string> {
+
+	const connectionProfile = getTargetConnectionProfile(
+		serverName,
+		azureResourceId,
+		username,
+		password,
+		// for login migration, when connecting to a target Azure SQL, use true/true
```

Comment:
for login migration scenario, verify that true/true is correct for the target (regardless of target platform)

Reply:
cc @AkshayMata

Reply:
Thanks for checking in! I love the way you clearly outlines all the scenarios and risk factors!

To answer your question, the doc above applies to "Azure SQL Database" and "Azure SQL Managed Instance." So for MI the default true/false will work. However, that doesn't work for SQL VM, which is why I had to hard code it in the first place. 

The long term solution should be:
- For SQL MI, default to true/false. 
- For SQL VM, we give customer UX dropdowns to select how they want to connect.